__init__.py

```diff
@@ -163,13 +146,42 @@ def initialize(self):
         # Support query for active alarms from other skills
         self.add_event("private.mycroftai.has_alarm", self.on_has_alarm)
 
+        # establish local timezone from config
+        self.local_tz = self.location_timezone
+        self.log.info("Local timezone configured for %s" % (self.local_tz,))
+
+        self.weekdays = None
+        self.months = None
```

Comment:
these should be initialized in the constructor.

Reply:
Done